pkg/config/configuration.go

```diff
@@ -275,6 +278,24 @@ func (m MetricSpec) GetHTTPIncreasedCardinality(log logger.Logger) bool {
 	return *m.HTTP.IncreasedCardinality
 }
 
+// GetLatencyDistribution returns a *view.Aggregration to be used for latency histograms
+func (m MetricSpec) GetLatencyDistribution(log logger.Logger) *view.Aggregation {
+	defaultLatencyDistribution := []float64{1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1_000, 2_000, 5_000, 10_000, 20_000, 50_000, 100_000}
+	log.Infof("metric spec: %v", m)
+	if m.LatencyDistributionBuckets == nil || len(*m.LatencyDistributionBuckets) == 0 {
```

Comment:
This means that a user can't disable adding latency buckets. That's fine, but want to be clear that's what we want to do.

Reply:
correct, it is just strange to have a histogram without buckets.  Maybe we could decide on a more sane default that the current very specific and long one.